libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.service.ts

```diff
@@ -46,6 +51,40 @@ export class NationalRegistryXRoadService {
     }
   }
 
+  async getNationalRegistryResidenceHistory(
+    nationalId: string,
+    authToken: string,
+  ): Promise<NationalRegistryResidenceHistory> {
+    const historyList = await this.nationalRegistryFetch<Array<Heimili>>(
+      `/${nationalId}/buseta`,
+      authToken,
+    )
+
+    const history = historyList.map((heimili: Heimili) => {
+      // API says Date, but is string -- fallback in case that changes in the future
+      const date =
+        typeof heimili.breytt === 'string'
+          ? new Date(heimili.breytt)
+          : heimili.breytt
+      return {
+        address: {
+          city: heimili.stadur,
+          postalCode: heimili.postnumer,
+          streetName: heimili.heimilisfang,
+        } as NationalRegistryAddress,
+        country: heimili.landakodi,
+        dateOfChange: date,
+      } as NationalRegistryResidence
+    })
+
+    computeCountryResidence(history)
```

Comment:
This function returns the history, but you are not using the outcome from it, why is that?